.github/workflows/ci-backend-2.11.yml

```diff
@@ -0,0 +1,36 @@
+name: Build 2.11
```

Comment:
This part (splitting into separate builds per Scala version) was just to figure out why Scala 2.13 build was failing. But, it made it pass. And bonus, makes them build in parallel. The downside is that we'll have to keep them up to date with Scala versions.